week-3/slideshow/style.css

```diff
@@ -1 +1,97 @@
-/** Write your CSS in here **/
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  height: 100vh;
+  display: grid;
+  justify-content: center;
+  font-family: monospace;
+  background: url(pexels-kseniya-lapteva-9175736.jpg)
+    no-repeat center;
+}
+.container {
+  position: relative;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-block: 3rem;
+}
+.heading {
+  text-align: center;
+  font-size: 2rem;
+}
+.img {
+  height: 25rem;
+  width: 50rem;
+  object-fit: cover;
+  box-shadow: 10px 10px 20px gray;
+}
+#backward-btn,
```

Comment:
If the same styling is used for multiple elements, which css selector is better choice - class or ID?